downstream/modules/platform/ref-general-inventory-variables.adoc

```diff
@@ -83,15 +143,18 @@ endif::aap-install[]
 ifdef::container-install[]
 For more information, see link:{URLContainerizedInstall}/ansible_automation_platform_containerized_installation#proc-set-registry-username-password[Setting registry_username and registry_password].
 endif::container-install[]
-
-| `registry_url` | `registry_url` | URL for the registry source. 
-
-Default = `registry.redhat.io`
-
-| `registry_username` | `registry_username` | Required if performing an online non-bundled installation.
-
-The username credential for access to the registry source defined in `registry_url`.
-
+| Required if `registry_auth=true` 
```

Comment:
`registry_auth` is a variable used in containerized installer only. For RPM installer, the var needs to be set if password is required to access `registry_url`.